src/appengine/handlers/jobs.py

```diff
@@ -41,7 +43,7 @@ def get_queues():
 
 
 class Handler(base_handler.Handler):
-  """Manage sets of environment variables for bots."""
+  """Return list of jobs and templates."""
```

Comment:
The new language seems a bit like it is for a comment instead of a class.

Reply:
Done